docs/flashbots-auction/searchers/advanced/rpc-endpoint-v1.mdx

```diff
@@ -232,7 +238,15 @@ example response:
 }
 ```
 
-### flashbots_getUserStats
+# Warning
```

Comment:
this is a title element -- should probably change to this style: 
```md
:::caution

...

:::
```

Reply:
done